eclipse-collections-code-generator/src/main/resources/impl/bag/immutable/immutablePrimitiveEmptyBag.stg

```diff
@@ -182,6 +184,14 @@ final class Immutable<name>EmptyBag implements Immutable<name>Bag, Serializable
         return this;
     }
 
+    @Override
+    public Immutable<name>Set selectUnique()
+    {
+        Immutable<name>Bag selected = this.selectByOccurrences(occurrences -> occurrences == 1);
+        Immutable<name>Set result = <name>Sets.immutable.ofAll(selected);
+        return result;
```

Comment:
Since this is an empty Bag, you can directly return `<name>Sets.immutable.empty()`